releasenotes/notes/1.3/clib-grover-op-cb032144e899ed0d.yaml

```diff
@@ -1,12 +1,12 @@
 ---
 features_circuits:
   - |
-    Added :func:`.grover_operator` to construct a Grover operator circuit, used in e.g.
-    Grover's algorithm and amplitude estimation/amplification. This function is similar to
-    :class:`.GroverOperator`, but does not require choosing the implementation of the
-    multi-controlled X gate a-priori and let's the compiler choose the optimal decomposition
-    instead. In addition to this, it does not wrap the circuit into an opaque gate and is 
-    faster as less decompositions are required to transpile.
+    Added :func:`.grover_operator` to construct a Grover operator circuit, used in 
+    Grover's algorithm and amplitude estimation/amplification, for example. This function is similar to
+    :class:`.GroverOperator`, but does not require you to choose the implementation of the
+    multi-controlled X gate, but instead lets the compiler determine the optimal decomposition.
+    Additionally, it does not wrap the circuit into an opaque gate and is 
+    faster because fewer decompositions are required for transpilation.
 
     Example::
```

Comment:
You can use a plot directive here so the circuit is actually shown in the release note. You just need to change the `print(grover_op.draw())` to `grover_op.draw("mpl")`